cobalt/build/configs/chromium_linux-x64x11/args.gn

```diff
@@ -9,3 +9,5 @@ chrome_pgo_phase = 0
 # E/GL(ES). Partners are only required to support upto GLES 2.0 : https://developers.google.com/youtube/cobalt/docs/reference/starboard/modules/16/gles#gles_version
 # Disable Vulkan until we formally decide to support it.
 enable_vulkan = false
+
+enable_widevine = true
```

Comment:
IDK if we want to build chromium builds with no cobalt changes with enable_widevine = true . That is not the default chromium build configuration:  https://chromium.googlesource.com/chromium/src/+/main/docs/linux/build_instructions.md#build-chromium

@kaidokert do we want to build chromium with this flag ? ( not sure if some chromium test targets depend on this )

Reply:
I'm assuming you want cobalt to build with enable_widevine = true for that you should now change this file: cobalt/build/configs/linux-x64x11/args.gn

Reply:
I added this to fix the broken linux CI builders (i've been developing on android which doesnt need this flag - its on by default - https://source.chromium.org/chromium/chromium/src/+/main:third_party/widevine/cdm/widevine.gni;l=15?q=enable_widevine&ss=chromium%2Fchromium%2Fsrc)
However I think the actual issue that is that i've missed an IS_STARBOARD guard. Let me investigate that and report back..

Reply:
removed.